src/index.html

```diff
@@ -136,5 +136,6 @@ <h4 class="data-text" id="increasing-stairs-container">2 Day Stair Streak</h4>
   </main>
   <script src="https://unpkg.com/packery@2.1/dist/packery.pkgd.min.js"></script>
   <script src="https://unpkg.com/draggabilly@2/dist/draggabilly.pkgd.min.js"></script>
+  <script type="text/javascript" src="src/scripts.js"></script>
```

Comment:
Since we are using webpack we do not need to link the script file in the the HTML - When wepback creates the bundle it added the script tag into the HTML - I believe this is set up in the webpack config file.